web/src/features/time/TimeAxis.tsx

```diff
@@ -58,33 +56,13 @@ const renderTick = (
             className="midnight-marker"
           />
         )}
-      <line stroke="currentColor" y2="6" opacity={shouldShowValue ? 0.5 : 0.2} />
-      {shouldShowValue &&
+      <line stroke="currentColor" y2="6" opacity={isMajorTick ? 0.5 : 0.2} />
+      {isMajorTick &&
         renderTickValue(value, index, displayLive, lang, selectedTimeRange, timezone)}
     </g>
   );
 };
 
-const isMidnight = (date: Date, timezone?: string) => {
```

Comment:
generalised this function into a `getLocalTime` method, which I moved to helpers.